datafusion/core/tests/sqllogictests/postgres/test_files/values_list.slt

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# values without casting
+# postgres
+#   1 2.0 -3 2
+#   10 20.0 -30 4
+# datafusion:
+#   1 2 -3 2
+#   10 20 -30 4
+
+query IRII
+SELECT * FROM
+(VALUES (1,2.0::real,-3,1+1),(10,20.0::real,-30,2+2))
```

Comment:
The original test doesn't have an explicit type cast. But Postgres treats `2.0` as a `numeric` type and prints it as `2.0`. Whereas in Datafusion it is `Float64` and the printed representation is `2`.

In the python comparison it didn't matter, because the values were compared. In sqllogictest text-based representations are compared.

Reply:
Results are now normalized before comparison.